DEVELOPMENT.md

```diff
@@ -0,0 +1,138 @@
+# Setup Development Environment
+
+## 1. Clone the repo
+
+```sh
+git clone --branch v2 https://github.com/aws/aws-node-termination-handler.git nthv2
+cd nthv2
+```
+
+## 2. Set environment variables
+
+*Tip:* Several steps in this guide, and utility scripts, use environment variables. Saving these environment variables in a file, or using a shell extension like [direnv](https://direnv.net), will make it easy to restore your development environment in a new shell instance.
+
+```sh
+export CLUSTER_NAME=<name>
+export AWS_REGION=<region>
+```
+
+## 3. Create a basic EKS Cluster
+
+Skip this set if you already have an EKS cluster.
+
+```sh
+envsubst <resources/eks-cluster.yaml.tmpl | eksctl create cluster --kubeconfig "${PWD}/kubeconfig" -f -
+
+export KUBECONFIG="$PWD/kubeconfig"
+```
+
+If you do not want to use `envsubst` you can copy the template file and substitute the referenced values.
+
+## 4. Create Infrastructure
+
```

Comment:
can you add a brief description to this section that lists all the resources being deployed?